packages/terra-form/docs/Textarea.md

```diff
@@ -0,0 +1,16 @@
+# Textarea
+
+Generic textarea which represents an HTML textarea element directly.
```

Comment:
We do want to expose it to consumers if they need it, but we recommend that they use TextareaField instead of this component.